source/robot_model/install.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+SOURCE_PATH=${PWD}
+
+apt-get update && apt-get install -y \
+    libeigen3-dev \
+    zlib1g-dev \
+    libssl-dev \
+    libboost-all-dev \
+    libboost-dev \
+    libboost-filesystem-dev \
+    libboost-system-dev \
+    libboost-chrono-dev \
+    libboost-date-time-dev \
+    libboost-thread-dev \
+    libboost-test-dev \
+    libbz2-dev \
+    libncurses-dev \
+    pax \
+    tar \
+    liburdfdom-dev \
+    libassimp-dev \
+    assimp-utils \
+    doxygen \
+    texlive-latex-extra
```

Comment:
Are some of these deps really necessary? for example texlive-latex-extra...

Reply:
they are actually, for openrobots I believe

Reply:
I don't like these dependencies ☹️ Do we really have to build pinocchio from source? why not use a precompiled package with apt? 
https://stack-of-tasks.github.io/pinocchio/download.html

Reply:
All of them are necessary yes. If I remember correctly, the precompiled where outdated by far when I was using them. I will check tomorrow if that has changed.